docs/decisions/0006-keep-eligibility-screener-and-recert-portal-separate.md

```diff
@@ -0,0 +1,41 @@
+# Keep the Eligibility Screener and the Participant Recertification Portal separate
+
+* Status: proposed
+* Deciders: @rocketnova, @microwavenby, ...
+* Date: [YYYY-MM-DD when the decision was last updated] <!-- optional -->
+
+Technical Story: https://wicmtdp.atlassian.net/browse/PRP-52
+
+## Context and Problem Statement
+
+Now that the [Eligibility Screener](https://github.com/navapbc/wic-mt-demo-project-eligibility-screener) is being re-written in Remix, the framework that we chose in [ADR-0005](0005-use-remix-for-project-architecture.md) to use for the Participant Recertification Portal,
+
+## Decision Drivers
+
+* Is there an easy way to keep the eligibility screener's dependencies up to date?
+* Is there an easy way to port improvements made in PRP to the eligibility screener?
+* Will it be easier or harder for WIC agencies to use or adopt these tools if they are combined into the same codebase?
+
+## Considered Options
+
+* Combine the eligibility screener and the recertification portal into the same codebase
+* Keep the eligibility screener and the recertification portal as two separate codebases
+
+## Decision Outcome
+
+...
+
+## Pros and Cons of the Options
+
+### Combine the eligibility screener and the recertification portal into the same codebase
+
+* Good, because it will be easy to keep the eligibility screener's dependencies up to date
+* Good, because we can more easily create and manage a library of shared React components
```

Comment:
```suggestion
* Good, because we can more easily share React components
```
I want to avoid the word Library when we mean Shared Code. IMO a library is a standalone component that both tools could pull in as a dependency

Reply:
I was actually thinking of that, a standalone unit of code that had the shared React components that the tools would pull in as a dependency. Do you think that's out of scope or not worth the effort?

Reply:
I think that having a library is a separate question than "should the screener and PRP be in the same repo", and mentioning it here as a "good" for the same codebase confuses things if you meant a separate library (it would apply to either option)